Few-iOS/Button.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Button.swift
+//  Few
+//
+//  Created by Coen Wessels on 13/03/15.
+//  Copyright (c) 2015 Josh Abernathy. All rights reserved.
+//
+
+import UIKit
+
+private class FewButton: UIButton {
```

Comment:
Does this exist just so the target-action mechanism works? Could we use `TargetActionTrampoline` instead?